pkgs/applications/networking/browsers/webbrowser/default.nix

```
@@ -0,0 +1,115 @@
+{ stdenv, lib, fetchgit, makeDesktopItem, pkgconfig, makeWrapper
+# Build
+, python2, autoconf213, yasm, perl, ccache
+, unzip, gnome2, gnum4
+
+# Runtime
+, xorg, zip, freetype, fontconfig, glibc, libffi
+, dbus, dbus-glib, gtk2, alsaLib, jack2, ffmpeg
+
+, withGTK3 ? false, gtk3
+}:
+
+let
+
+  libPath = lib.makeLibraryPath [ ffmpeg ];
+  gtkVersion = if withGTK3 then "3" else "2";
+
+in stdenv.mkDerivation rec {
+  pname = "webbrowser";
+  version = "29.0.0rc1";
+
+  src = fetchgit {
+    url = "https://git.nuegia.net/webbrowser.git";
+    rev = "fe8c1f5bb36bb5012bc72e6ee0b0b4d24c82619d";
```

```suggestion
    rev = version;
```
tags are preferred. I see that 29.0.0rc1 is 4 commits ahead of fe8c1f5bb36bb5012bc72e6ee0b0b4d24c82619d but that should be fine right?

---

Hello, thank you for you attention !
First, this isn't palemoon but another project, I just based my derivation on the palemoon derivation, and it seems I forgot to change the website.
Secondly, the tag wasn't here before, I just suggested today to the code's author to add them so that it becomes easier to maintain, and it seems he did this also today. So, yes, thank you for seeing that.
And finally, I will add myself to maintainers-list.nix, I didn't know that this existed.